docs/standard/data/sqlite/backup.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Online Backup
+title: Online backup
 ms.date: 12/13/2019
 description: Learn how to use SQLite's online backup feature.
 ---
-# Online Backup
+# Online backup
 
-SQLite can back up database files while the app is running. This functionality is available in Microsoft.Data.Sqlite as the [BackupDatabase](/dotnet/api/microsoft.data.sqlite.sqliteconnection.backupdatabase) method on SqliteConnection.
+SQLite can back up database files while the app is running. This functionality is available in `Microsoft.Data.Sqlite` as the <xref:Microsoft.Data.Sqlite.SqliteConnection.BackupDatabase%2A> method on `SqliteConnection`.
 
 [!code-csharp[](../../../../samples/snippets/standard/data/sqlite/BackupSample/Program.cs?name=snippet_Backup)]
 
-Currently, BackupDatabase will back up the database as quickly as possible blocking other connections from writing to the database. Issue [#13834](https://github.com/aspnet/EntityFrameworkCore/issues/13834) would provide an alternative API to back up the database in the background allowing other connections to interrupt the backup and write to the database. If you're interested, please provide feedback on the issue.
+Currently, `BackupDatabase` will back up the database as quickly as possible blocking other connections from writing to the database. Issue [#13834](https://github.com/aspnet/EntityFrameworkCore/issues/13834) would provide an alternative API to back up the database in the background allowing other connections to interrupt the backup and write to the database. If you're interested, provide feedback on the issue.
```

Comment:
We try to avoid backticks in the EF docs since they’re visually distracting. We try to use them sparingly to call attention to important parts of a paragraph (similar to using bold). Happy to change that in these docs, just voicing our philosophy.

Reply:
I don't like the styling either but this is our philosophy because it's better for localization, especially machine translated articles, which might be the case here. There's another option that we could pursue which is to use a new metadata called no-loc: https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#non-localized-strings